scout/server/blueprints/panels/templates/panels/panels.html

```diff
@@ -27,6 +28,48 @@
   </div>
 {% endblock %} <!-- end of content_main -->
 
+
+
+ 
+<!-- Create a Bootstrap card containing text bar and submit button for searching -->
+{% macro search_gene_card() %}
+<div class="container-float">
+  <div class="card mt-3">
+    <div class="card-body">
+      <form action="{{ url_for('panels.panels') }}" enctype="multipart/form-data" method="POST">
+        <div class="row">
+          <div class="col-5">
+            <input type="text" name="search_for" class="form-control" placeholder="Search Gene in Panels" required
+                   pattern="^[a-zA-Z0-9_\-]*$" title="Only alphanumeric characters (A-Z+a-z+0-9) and underscores allowed.">
+          </div>
+          <div class="col">
+            <button type="submit" class="btn btn-secondary">Search!</button>
+          </div>
+          <div class="col">
+          </div>
+      </form>
+
+      <div id="SearchResults" class="card-body">
+        {% if search_result %}
+        <strong> <em>{{search_string}}</em> found in: </strong> <br>
+        {% for name, version in search_result %}
```

Comment:
Search results are quite difficult to read, especially if a gene is found on many panels and panel versions:

<img width="709" alt="image" src="https://user-images.githubusercontent.com/28093618/183579781-53e92f35-7a12-49ab-bd8a-e9ebc11e7a6a.png">

Would be nice to apply some formatting to have these results in a table or something

Reply:
Done!